src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -80,10 +87,15 @@ public void resetGryo() {
         gyro.reset();
     }
 
+    /**
+     * 
+     * @return the position of the robot on the field as x and y in meters
```

Comment:
a Pose2d includes both position and heading

Reply:
what's heading?

Reply:
oh, wait nvm